Core/include/Acts/SpacePointFormation2/StripSpacePointBuilder.hpp

```diff
@@ -0,0 +1,87 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Units.hpp"
+
+#include <optional>
+
+namespace Acts {
+
+class GeometryContext;
+class Surface;
+
+namespace StripSpacePointBuilder {
+
+struct ClusterPairingOptions {
+  /// vertex position
+  Vector3 vertex = Vector3::Zero();
+  /// Accepted distance between two clusters
+  double maxDistance = 100 * UnitConstants::mm;
+  /// Accepted squared difference in theta for two clusters
+  double maxAngleTheta2 = 1;
```

Comment:
Those values seem a bit arbitrary, I guess they are from the legacy code? Would it make more sense to give the non squared values with unit as default?